src/native/libs/System.Security.Cryptography.Native/configure.cmake

```diff
@@ -29,6 +29,16 @@ check_function_exists(
     HAVE_OPENSSL_SHA3_SQUEEZE
 )
 
+check_function_exists(
+    EVP_rc2_ecb
+    HAVE_OPENSSL_EVP_RC2_ECB
```

Comment:
Do we need a new define here? Wouldn't this already be reflected by an OSSL define (NO_RC2, I think)?

Reply:
We can use `opensslconf.h` which defines `OPENSSL_NO_RC2`. I didn't want to do that for two reasons.

1. We don't currently import the header that tells us that, and I am not sure when it was introduced in OpenSSL.
2. We flip defines to "true" in portable builds. I don't know if we can or should re-define something that we didn't initially define ourselves.

Reply:
I guess we can meet in the middle and define ours based on the OpenSSL define.